views/pages/toplevel/home.njk

```diff
@@ -48,9 +48,8 @@
                     <span class="u-wide-only">{{ copy.callsToAction.under10k.long }}</span>
                     <span class="u-not-wide">{{ copy.callsToAction.under10k.short }}</span>
                 </span>
-            </a>
-
-            <a class="btn btn--medium
+            </a><!-- Fix inline-block spacing
+            --><a class="btn btn--medium
```

Comment:
I'm not a fan in general of this technique but understand why we're using it – if we need to do it anywhere else though maybe let's use flexbox or similar instead?